app/models/allocation.rb

```diff
@@ -48,7 +48,6 @@ class Allocation < VersionedModel
   belongs_to :from_location, class_name: 'Location'
   belongs_to :to_location, class_name: 'Location'
   has_many :moves, inverse_of: :allocation, dependent: :destroy, autosave: true
-  has_many :events, as: :eventable, dependent: :destroy
```

Comment:
Should this be replaced with generic_events rather than being removed entirely? It looks like we only create allocation events on the underlying moves currently when the allocation is cancelled, but adding move level events might be something we want to do in future. I guess it's fine to remove this until that requirement surfaces 🤔

Reply:
We're not referencing the relationship. Am happy to add it in if we want it to flag this up in the code

Reply:
Hang on, this is actually wrong. Moves have events not Allocations

Reply:
Allocations cancel calls iterate through the Allocation#moves and generate MoveCancel events for them. So this was actually always wrong

Reply:
Agreed 👍